src/fragments/lib/ssr/js/getting-started.mdx

```diff
@@ -75,6 +75,38 @@ Server-side functions that don't have a req object (e.g. Next.js' getStaticProps
 
 </Callout>
 
+## Next.js App Router Usage
+Amplify JS can be used with Next.js' App Router with a few important changes to how you configure and use the library, namely:
+
+- Amplify JS must be explicitly configured in both your client-side and server-side code (with `ssr` enabled)
+- An appropriate request object will need to be prepared for use with `withSSRContext` when performing operations on the server that require authentication
+
+Example of an Next.js App Router component that uses Amplify JS on the server:
+
+```js
+import { Amplify, withSSRContext } from 'aws-amplify';
+import { headers } from 'next/headers';
+import { listPosts } from '@/graphql/queries'; 
+import awsExports from '@/aws-exports';
+
+Amplify.configure({ ...awsExports, ssr: true });
+
+export default async function Page() {
+  // Construct a req object & prepare an SSR enabled version of Amplify
+  const req = {
+    headers: {
+      cookie: headers().get('cookie'),
+    },
+  };
+  const SSR = withSSRContext({ req })
+
+  const { data } = await SSR.API.graphql({ query: listPosts });
```

Comment:
Could be more clear if we use `Auth.currentAuthenticatedUser()` instead?

Reply:
+1 that might be a simpler use case to show, and one that we get questions about!

Reply:
Hum I'm open to that, I just used `graphql` with `listPosts` here to better align with the other examples on the page. What do you think?

Reply:
I think we should leave it as the same example for now. This page in general will be broken apart into each seperate category in the near future.